tools/deb/istio-iptables.sh

```diff
@@ -86,62 +99,68 @@ while getopts ":p:u:e:i:h" opt; do
   esac
 done
 
-
-# TODO: more flexibility - maybe a whitelist of users to be captured for output instead of
-# a blacklist.
-if [ -z "${ENVOY_UID:-}" ]; then
+# TODO: more flexibility - maybe a whitelist of users to be captured for output instead of a blacklist.
+if [ -z "${PROXY_UID}" ]; then
   # Default to the UID of ENVOY_USER and root
-  ENVOY_UID=$(id -u ${ENVOY_USER:-istio-proxy})
+  PROXY_UID=$(id -u ${ENVOY_USER:-istio-proxy})
   if [ $? -ne 0 ]; then
-     echo "Invalid istio user $ENVOY_UID $ENVOY_USER"
+     echo "Invalid istio user $PROXY_UID $ENVOY_USER"
      exit 1
   fi
   # If ENVOY_UID is not explicitly defined (as it would be in k8s env), we add root to the list,
   # for ca agent.
-  ENVOY_UID=${ENVOY_UID},0
+  PROXY_UID=${PROXY_UID},0
 fi
 
 # Remove the old chains, to generate new configs.
-iptables -t nat -D PREROUTING -p tcp -j ISTIO_INBOUND 2>/dev/null
```

Comment:
This will result in an unexpected message the first time it is run. ( and next lines - the second time it is run ).

I think ignoring the err is useful and avoids confusion

Reply:
@costinm that's fine .... doing this seems to negate the `set -x` at the top of the file, resulting in the echo for each statement also being redirected to `/dev/null`.  You you know offhand some bash magic to make it work?